### DISCUSSION TITLE
Undefined control sequence related with \ntaddfile

### DISCUSSION THREAD

**Discussion by pedromsrocha** [joaomlourenco/novathesis#205] [Q&A] [answered]
Hi, 

I'm having the following compilation error 

>(./Config/6_files.tex
>NT FILE Config/6_files.tex
>! Undefined control sequence.
>\@nt@addtolist@iv .../@nt/folders}/#4}}}\expanded 
>                                                  {\options {/@nt/#1/#2/#3/....
>l.23 \ntaddfile{bib}{bibliography.bib}

**Comment by joaomlourenco** [comment, marked as answer]
Does it happen with the "original" (unmodified) template as downloaded from the repository?

YES:  please update your LaTeX distribution.
NO:  please provide more info for helping in the debug process, e.g., comment that line "\ntaddfile{bib}{bibliography.bib}" and let us know if the error persists.

**Comment by pedromsrocha** [comment]
I've updated my LaTeX and that fixed the issue. Thank you.
